sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-full-time-process-response-integration.scheduler.e2e-spec.ts

```diff
@@ -0,0 +1,280 @@
+        {
+          processSummary: [
+            `Processing file ${MSFAA_FULL_TIME_RECEIVE_FILE_WITH_REACTIVATION_RECORD}.`,
+            "File contains:",
+            "Confirmed MSFAA records (type R): 1.",
+            "Cancelled MSFAA records (type C): 0.",
+            "Record from line 1, updated as confirmed.",
+          ],
+          errorsSummary: [],
+        },
+      ]);
+      // Assert that the file was deleted from SFTP.
+      expect(sftpClientMock.delete).toHaveBeenCalled();
+      // Find the updated MSFAA records previously created.
+      const msfaaIDs = createdMSFAARecords.map((msfaa) => msfaa.id);
+      const [createdReactivatedMsfaaRecord] = createdMSFAARecords;
+      const msfaaUpdatedRecords = await db.msfaaNumber.find({
+        select: {
+          msfaaNumber: true,
+          dateSigned: true,
+          serviceProviderReceivedDate: true,
+          cancelledDate: true,
+          newIssuingProvince: true,
+        },
+        where: {
+          id: In(msfaaIDs),
+        },
+        order: {
+          msfaaNumber: "ASC",
+        },
+      });
+      expect(msfaaUpdatedRecords).toHaveLength(msfaaInputData.length);
+      const [reactivatedMSFAA] = msfaaUpdatedRecords;
+      // Validate reactivated confirmed record.
+      expect(reactivatedMSFAA.dateSigned).toBe("2023-05-02");
+      expect(reactivatedMSFAA.serviceProviderReceivedDate).toBe("2023-05-03");
+      expect(reactivatedMSFAA.cancelledDate).toBe(null);
+      // Validate msfaa updated to the reactivated msfaa for all pending disbursements.
+      const [appAFirstDisbursementSchedule, appASecondDisbursementSchedule] =
+        applicationA.currentAssessment.disbursementSchedules;
+      const appAFirstDisbursementScheduleMsfaaNumberId =
+        await db.disbursementSchedule.findOne({
+          select: {
+            msfaaNumberId: true,
+          },
+          where: { id: appAFirstDisbursementSchedule.id },
+        });
+      const appASecondDisbursementScheduleMsfaaNumberId =
+        await db.disbursementSchedule.findOne({
+          select: {
+            msfaaNumberId: true,
+          },
+          where: { id: appASecondDisbursementSchedule.id },
+        });
+      const [appBFirstDisbursementSchedule, appBSecondDisbursementSchedule] =
+        applicationB.currentAssessment.disbursementSchedules;
+      const appBFirstDisbursementScheduleMsfaaNumberId =
+        await db.disbursementSchedule.findOne({
+          select: {
+            msfaaNumberId: true,
+          },
+          where: { id: appBFirstDisbursementSchedule.id },
+        });
+      const appBSecondDisbursementScheduleMsfaaNumberId =
+        await db.disbursementSchedule.findOne({
+          select: {
+            msfaaNumberId: true,
+            disbursementScheduleStatus: true,
+          },
+          where: { id: appBSecondDisbursementSchedule.id },
+        });
+      // Validate msfaa updated for all pending disbursements.
```

Comment:
thought the comment and below code does not match, I think after https://github.com/bcgov/SIMS/pull/2010/files#r1230097110 fix it will be good

Reply:
Yes, this confused me too.